sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/protobuf/SymbolRefsImporter.java

```diff
@@ -48,6 +49,33 @@ public SymbolRefsImporter(SensorContext context, UnaryOperator<String> toRealPat
     this.context = context;
   }
 
+  private static void addReferences(InputFile file, SymbolReferenceInfo.SymbolReference tokenInfo, NewSymbolTable symbolTable) {
+    var declarationRange = toTextRange(file, tokenInfo.getDeclaration());
+    if (declarationRange.isPresent()) {
+      NewSymbol symbol = symbolTable.newSymbol(declarationRange.get());
+      for (SonarAnalyzer.TextRange refTextRange : tokenInfo.getReferenceList()) {
+        var validatedReference = validatedReference(file, refTextRange, declarationRange.get());
+        validatedReference.ifPresent(symbol::newReference);
+      }
+    }
+  }
+
+  private static Optional<TextRange> validatedReference(InputFile file, SonarAnalyzer.TextRange refTextRange, TextRange declarationRange) {
+    var referenceRange = toTextRange(file, refTextRange);
+    if (referenceRange.isEmpty()) {
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("The reported token was out of the range. File {}, Range {}", file.filename(), refTextRange);
```

Comment:
We lost coverage for that line because in the new pb, there is not "out of range" anymore (either fixed in .Net 8 SDK or fixed somewhere else).

Reply:
They fixed the tab problem and we used tabs to reproduce that out-of-range issue.